packages/next/kotti-field-radio-group/KtFieldRadioGroup.vue

```diff
@@ -75,15 +78,20 @@ export default defineComponent({
 		const { forceUpdate, forceUpdateKey } = useForceUpdate()
 		const name = ref(`autoGeneratedRadioName${nameIndex++}`)
 
+		const
+
 		return {
 			inputProps: computed(() => ({
 				...field.inputProps,
-				class: 'kt-field-radio-group__wrapper__input',
 				name,
 				type: 'radio',
 			})),
 			field,
 			forceUpdateKey: forceUpdateKey.value,
+			wrapperClasses: computed(() => ({
```

Comment:
Please don’t insert things into random places. Lexicographic ordering is easier to read and for finding things later.